Updated the installation guide (#149)
ta-oliver/infertrade: conda_installation_walkthrough_for_infertrade/installation_guide/installation_guide.md

![](images/infertrade_logo.png)

# Welcome!

**New to Anaconda?**
**No problem, let's take a brief look into it**

### Why would the team at Infertrade make documentation regarding this?
* If you work with Data or Machine Learning then you might be fond of Anaconda software. It is great as it provides a flexible environment which makes your job faster and easier.

* Let’s say, you already have multiple versions of python installed and you want to use Python 3.7x for our product but currently, you are using python 3.9x in your project. In that case. You will experience some difficulties in creating a virtual environment with Python 3.7x using virtualenv, venv or any other tool except Conda. 

> * In case you somehow manage to install it without using Conda you might still experience problems while installing some of the dependencies.

Reviewer: ```suggestion
* If you install it without using Conda or virtualvenv you might still experience problems while aligning some of the dependencies.
```

[Anaconda Download](https://www.anaconda.com/products/individual "Install anaconda")

AFTER INSTALLING ANACONDA

*Below is the link to a Conda guide used for dealing with the virtual environment provided by Conda.*

[conda-cheatSheet](https://docs.conda.io/projects/conda/en/4.6.0/_downloads/52a95608c49671267e40c689e0bc00ca/conda-cheatsheet.pdf "conda-cheatsheet")

**Let’s create a virtual environment with python 3.7x using Conda, we went with the name – "infertradenv", but are always free to set the name of your choice.**

* Step 1: Open Anaconda prompt and navigate to your desired drive.

* Step 2: Make a sub folder called “infertrade” in your user’s folder, and then use the cd infertrade command.

# Follow the commands used in the following steps

* Step 3: conda create --name infertradenv python = 3.7

* Step 4: conda env list

* Step 5: conda activate infertradenv

* Step 6: python --version - This is to make sure you are using python 3.7

* Step 7: curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py

* Step 8: py get-pip.py

* Step 9: pip install infertrade

* Step 10: pip install seaborn

* Step 11: pip install Jupyter notebook

> * Step 12: Open Jupyter notebook in you're activated virtual environment by running the command in anaconda prompt  (Jupyter notebook)

Reviewer: ```suggestion
* Step 12: Open [Jupyter notebook](https://jupyter.org/) in your activated virtual environment by running the command in the Anaconda prompt: `jupyter notebook`
```

pip install jupyter notebook

> Jupter notebook

Reviewer: ```suggestion
jupyter notebook
```
